sympy/ntheory/factor_.py

```diff
@@ -993,11 +993,11 @@ def factorint(n, limit=None, use_trial=True, use_rho=True, use_pm1=True,
         factordict = n
     if factordict and (isinstance(n, Mul) or isinstance(n, dict)):
         # check it
-        for k in list(factordict.keys()):
-            if isprime(k):
+        for key in list(factordict.keys()):
```

Comment:
Did LGTM complain about this, but not `k, v in factordict.items()` below? I would change LGTM on this one. ;-) Alternatively, perhaps use `m` to be like `n` in this context. But really, what's the LGTM complaint?

Reply:
Oh...duplicated loop variables. LGTM should be smart enough (IMO) to not complain about duplication unless it is in the same loop context. The loop below uses k, too, but it is not in the scope of the first loop (unlike the case above with throw-away `i` being used again within the loop.

Reply:
It did complain about the one below, but this seemed like a better way to solve it.